token/users.go

```diff
@@ -0,0 +1,66 @@
+package token
+
+import (
+	"encoding/json"
+	"io/ioutil"
+	"net/http"
+	"net/url"
+	"path/filepath"
+
+	"github.com/fabric8-services/fabric8-tenant/configuration"
+	"github.com/pkg/errors"
+)
+
+type userData struct {
+	Data struct {
+		Attributes struct {
+			Cluster string `json:"cluster,omitempty"`
+		}
+	}
+}
+
+type UserProfileService interface {
+	GetUserCluster(userID string) (string, error)
+}
+
+type UserController struct {
```

Comment:
Call it a `UserProfileClient` . We reserve the usage of `Controller`.. only for the Goa request controllers.

Reply:
done!